Doc/whatsnew/3.8.rst

```diff
@@ -284,6 +284,8 @@ Optimizations
   syscalls is reduced by 38% making :func:`shutil.copytree` especially faster
   on network filesystems. (Contributed by Giampaolo Rodola' in :issue:`33695`.)
 
+* :func:`shutil.copytree` now accepts a new ``dirs_exist_ok`` keyword argument.
+  (Contributed by Josh Bronson in :issue:`20849`.)
```

Comment:
You listed this change under the "Optimization" section. It's not. Add it to "Improved Modules" section instead.

Reply:
Done in the latest revision.